stories/Toggle/Toggle.stories.jsx

```diff
@@ -0,0 +1,41 @@
+import Toggle from '../../components/Toggle';
+
+const Template = args => {
+  const { label, labelPosition, id, ...rest } = args;
+  return (
+    <div style={{ display: 'flex', alignContent: 'center' }}>
+      {labelPosition === 'left' && (
+        <label htmlFor={id} style={{ marginRight: 10 }}>
+          {label}
+        </label>
+      )}
+      <Toggle id={id} {...rest} />
+      {labelPosition === 'right' && (
+        <label htmlFor={id} style={{ marginLeft: 10 }}>
+          {label}
+        </label>
+      )}
+    </div>
+  );
+};
```

Comment:
I don't think that is interesting includes extras elements on component documentation. We could improve it with a new prop called label and its position, but this can be fixed in another task. What do you think?

Reply:
I think it's super valid, in the construction of the template I found this extra element of the label strange